### PR TITLE
Updated preprocessing steps

### PR DESCRIPTION
**Pull request by JavierACM**
Updates:
- Condensed read data functions into one.
- Updated test_data_ingestion.py to reflect changes.
- Created EDA folder.
- Moved eda_report.py.
- Deleted EDA HTML reports.

Future work:
- Replace eda_report.py with Jupyter notebook.
- Display EDA report within above-mentioned notebook.
- Read ERCOT market data.
- Create raw data ERD.

### CODE REVIEW

File: src/data_ingestion.py [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
-    data = pd.read_csv(path)
-    return data
-
-# Read status data
-def read_status():
-    '''Function to process the status data'''
-    path = RAW_PATH + 'status_data.csv'
-    data = pd.read_csv(path)
-    return data
+def read_data(name):
```
[the remark this answers is not in the document]
Comment (JavierACM) [reply src/data_ingestion.py]: Thank you for the feedback! I never used type hints before, but I'll be implementing them. 😄

File: src/data_ingestion.py [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
+    Returns
+        data: DataFrame
+            The data loaded from the shared data file.
+    '''
+    
+    # Define valid names
+    valid = ['fault', 'scada', 'status']
+
+    # Test if arg is a valid name
+    if name in valid:
```
[the remark this answers is not in the document]
Comment (JavierACM) [reply src/data_ingestion.py]: I was in between using try & except and the code I currently have. I'll switch over to using try / except. I'll also look into logging; it's a new concept for me. Thank you!

**Comment by JavierACM** [comment]
- Added type hints to data_ingestion functions
- Added FileNotFoundError exception handling
- Changed naming for datasets
- Converted test_data_ingestion file into Jupyter Notebook

**Review by rr-85 — APPROVED** [review]
I liked the notebook structure, I was successful able to run the notebook in turbine_env conda environment create by .yml file. I was got the merger_data.csv as an output for running the function.

File: src/data_ingestion.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ntuziyihuang) [reply src/data_ingestion.py]: @JavierACM I like your idea of checking the valid data name before implementing the function. Meanwhile using try/except can definitely improve the code and inform what errors are raised. I think later we can also think about using what Error alert can better guide the user.

**Review by ntuziyihuang — COMMENTED** [review]
Overall, it is very clear and readable.

**Pull request merged**